---
author: roddy
comments: true
date: 2018-07-05 14:00:00
layout: post
slug: phd_studentship_lido_bbsrc
title: "Open PhD studentship:  Data science & machine learning for genomic analysis"
categories:
 - labnews
---


**Interested in supercharging the productivity of genome biologist researchers?**

We have an exciting 4-year bioinformatics PhD position open through the [London BBSRC LiDO Doctoral Training Programme](http://lido-dtp.ac.uk/npif2018.html). 

[**Apply by 5pm July 20th** here at LIDO](http://lido-dtp.ac.uk/npif2018.html) to start in September.

A description of the project is below. It is *highly* interdisciplinary - no need to already be able to understand all the details today. 

Great candidates fulfill 3 of the following four criteria: smart, hard working, understands genomes, and not scared of data analysis or coding. 

If you have any questions regarding scope or nature of the project, or whether your skills are potentially sufficient, please don't hesitate to get in touch with me (Yannick).

(Standard UKRI eligibility criteria apply (i.e. I *think* one must be UK resident). - the LiDO people can explain this better). 


## Project summary

(apologies for the use of domain-specific jargon!)

<div class="card" style="max-width: 664px; margin:5px auto;">
  <div class="card-body">
    <h3 class="card-title">Inferring gene function for emerging model organisms</h3>
	<p class="card-text">The first generation of molecular-genetic research focused on traditional model organisms including mouse, yeast, zebrafish, Drosophila, and C. elegans. Genetic research increasingly uses diverse organisms that are much more relevant models for specific questions. For example, some such emerging organisms exhibit unique phenotypes including 100-fold intra-specific variation in lifespan, resistance to harsh environmental conditions, represent novel animal models for disease or development, provide crucial ecosystem services, or are key to food security because they are crops or may pollinate them.</p>
 
<p> A major challenge when working with such “emerging” model organisms is making sense of the “gene lists” that result from genome-wide analyses (e.g., of gene expression or genome-wide associations).</p>
 
<p>Here, we will develop a bioinformatics tool that takes a list of genes or genomic locations from a new species as input, and transparently produces relevant functional information describing this list of loci. When presented with data for which no direct information exists, the tool will in a first instance identify relationships of orthology to regions of other species. This will create a trail of links to databases in which functional information for orthologous regions does exist. These databases will be interrogated following hierarchical set of rules (initially defined based on human-curated examples). Using using cutting-edge “learning to rank” machine learning techniques the rulesets will be refined over time by tracking user behaviour (based on logs of which relationships/trails users retain) as well as explicitly allowing users to flag issues. The tool hereby makes it possible to extract significant value from largescale datasets that would otherwise require laborious case-by-case engineering efforts to connect. Summary data will be returned to the user using visualisations, statistics and tables in a manner that facilitates interpretation. Inferences and relationship calculations taking seconds will be available immediately; those taking minutes (e.g., distant orthology) will appear asynchronously as they complete; and those taking longer will result in email notification.</p>

<p>We will package our work in a manner that makes it accessible to biologists working with new or existing genomes. This builds on our extensive success with including with the <a href="https://www.sequenceserver.com">SequenceServer</a> and OMA software. Overall, our approach will substantially improve the ability of genome biologists to generate meaningful biological insight when working with new organisms. </p>

  </div>
</div>




This project is in collaboration with [Christophe Dessimoz](http://lab.dessimoz.org) at UCL/Lausanne. 
